README.md

```diff
@@ -452,6 +453,36 @@ const main = async () {
 
 --------------------------------------------------------
 
+<a name="import"></a>
+ES6 Import and Typing(for TypeScript)
+-------------------------------------
+
+We have two ways to import(require) the levelup module in the code.
+
+### 1. The old fashioned `require`
+
+```js
+var levelup = require('levelup')
+```
+
+### 2. The new ES6 `import`
+
+```js
+import levelup from 'levelup'
+```
+
+## TypeScript
```

Comment:
Maybe remove this section? Not sure what it has to do with this PR? (And we did remove typings for now until a later version)